azext_edge/tests/edge/orchestration/resources/dataflow_endpoint/helper.py

```diff
@@ -0,0 +1,157 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
```

Comment:
This helper module (should it not be plural?) is not my favorite location for placement of `get_dataflow_endpoint_endpoint(...)` and `get_mock_dataflow_endpoint_record(...)`. One reason, it is not consistent with other orchestration resource test modules. It seems one off, a pattern like this only works if you have subpackage/nested directory. Usage in the clone test module is an illustration of this, where every get_endpoint and get_record import are uniformly imported but for dataflow endpoint it gets imported from dataflow_endpoint.helper.